floss/main.py

```diff
@@ -537,7 +537,11 @@ def main(argv=None) -> int:
     # can throw away result later if not desired in output
     time0 = time()
     interim = time0
+
     static_strings = get_static_strings(sample, args.min_length)
+    if static_strings == []:
+        return 1
```

Comment:
i don't think this should be treated as an error. both of us can trivially construct a non-empty file with 0 ASCII strings and i think FLOSS should be able to handle these. 

printing a logging message might be fine but let's not return an error status code.